tests/plugins/button/_helpers/buttontools.js

```diff
@@ -0,0 +1,94 @@
+window.buttonTools = {
+	// Standard aria attributes for button element.
+	typicalButtonAttributes: {
```

Comment:
Instead of the comment, we might rename the property  to:
```suggestion
	buttonAriaAttributes: {
```

or even something longer
```suggestion
	buttonStandardAriaAttributes: {
```

When I first read it - I wonder what are the `typical` attributes.